pages/index.tsx

```diff
@@ -1,166 +1,48 @@
-                  </svg>
-                </a>
-              </li>
-              <li>
-                <a href="https://www.linkedin.com/in/benjamin-beugnet-web-services/" target="_blank" >
-                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-11 19h-3v-11h3v11zm-1.5-12.268c-.966 0-1.75-.79-1.75-1.764s.784-1.764 1.75-1.764 1.75.79 1.75 1.764-.783 1.764-1.75 1.764zm13.5 12.268h-3v-5.604c0-3.368-4-3.113-4 0v5.604h-3v-11h3v1.765c1.396-2.586 7-2.777 7 2.476v6.759z"/></svg>
-                </a>
-              </li>
-              <li>
-                <a href="https://www.facebook.com/benjaminbeugnetws" target="_blank" >
-                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M22.675 0h-21.35c-.732 0-1.325.593-1.325 1.325v21.351c0 .731.593 1.324 1.325 1.324h11.495v-9.294h-3.128v-3.622h3.128v-2.671c0-3.1 1.893-4.788 4.659-4.788 1.325 0 2.463.099 2.795.143v3.24l-1.918.001c-1.504 0-1.795.715-1.795 1.763v2.313h3.587l-.467 3.622h-3.12v9.293h6.116c.73 0 1.323-.593 1.323-1.325v-21.35c0-.732-.593-1.325-1.325-1.325z"/></svg>
-                </a>
-              </li>
-            </ul>
-          </li>
-          <li>
-            <Image
-              src={mBalloisPic}
-              alt={t('landing-member-2-alt')}
-              width={250}
-            />
-            <h3 className={bigJohnPro.className}>Margot Ballois</h3>
-            <ul className={styles.network}>
-              <li>
-                <a href="https://www.linkedin.com/in/margot-ballois/" target="_blank" >
-                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M19 0h-14c-2.761 0-5 2.239-5 5v14c0 2.761 2.239 5 5 5h14c2.762 0 5-2.239 5-5v-14c0-2.761-2.238-5-5-5zm-11 19h-3v-11h3v11zm-1.5-12.268c-.966 0-1.75-.79-1.75-1.764s.784-1.764 1.75-1.764 1.75.79 1.75 1.764-.783 1.764-1.75 1.764zm13.5 12.268h-3v-5.604c0-3.368-4-3.113-4 0v5.604h-3v-11h3v1.765c1.396-2.586 7-2.777 7 2.476v6.759z"/></svg>
-                </a>
-              </li>
-            </ul>
-          </li>
-        </ul>
-      </section>
+      <>
+        <h1>{t("main-title")}</h1>
+        <div className={styles.sectionsList}>
```

Comment:
ul instead of div box
Then you can put an li to wrap all HomeSection components